test/e2e/create_test.go

```diff
@@ -385,3 +408,91 @@ func (tc *testContext) listFullyConfiguredWindowsNodes(isBYOH bool) ([]v1.Node,
 	}
 	return windowsNodes, nil
 }
+
+// TODO: vSphere CSRs are not being approved due to a hostname mismatch. This is an issue only because the
```

Comment:
Nit: this could have been added to a separate package. which we then could have deleted.